app/page.tsx

```
@@ -1,22 +1,35 @@
 'use client'
 
-import { useEffect, useState } from 'react'
+import { useEffect, useRef } from 'react'
 import { useRouter } from 'next/navigation'
-import Link from 'next/link'
-import Image from 'next/image'
+import Head from 'next/head'
```

ESLint

[error] 'Head' is defined but never used.

(@typescript-eslint/no-unused-vars)